content/install/_index.md

```diff
@@ -62,9 +62,19 @@ The available packages are:
 
 * [flatpak](https://www.flathub.org/apps/details/org.darktable.Darktable)
 
+<h2 id='windows'>Windows</h2>
+
+The installer supports Windows 8.1 w/ [UCRT](https://support.microsoft.com/en-us/topic/update-for-universal-c-runtime-in-windows-c0514201-7fe6-95a3-b0a5-287930f3560c) added, and later.
+
+### MSYS2 Package Manager
+
+From your UCRT64, CLANG64, or CLANGARM64 terminal run:
+
+    pacman -S --needed ${MINGW_PACKAGE_PREFIX}-darktable
```

Comment:
Actually, just realized this might not be built against the same version of libraries (nor toolchain) as the "official" installer, so it's fine if we don't want to recommend/support this.

Same goes for MacPorts & Homebrew installations below btw.

Instead we could mention other Windows package managers like winget, Chocolatey, and Scoop - they all seem to pull down and install the official installer directly from the github release links anyway...